### PR TITLE
[RFC][Pl 915] Add service skeletons

### PR DESCRIPTION
**Pull request by hfcorreia**

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+type Config struct {
+	SourceTable       string `mapstructure:"source-table"`
+	TargetTable       string `mapstructure:"target-table"`
+	SourceProfile     string `mapstructure:"source-profile"`
+	TargetProfile     string `mapstructure:"target-profile"`
+	ReadProvisioning  int64  `mapstructure:"read-units"`
+	WriteProvisioning int64  `mapstructure:"write-units"`
+}
+
```
Comment (raffo0707) [inline config.go]: missing comment

File: config.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package dynamodbcopy
+
+import "github.com/spf13/viper"
+
+type Config struct {
```
Comment (raffo0707) [inline config.go]: missing comment

File: config.go [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+	SourceTable       string `mapstructure:"source-table"`
```
Comment (raffo0707) [inline config.go]: what a combo!!!

File: dynamodb.go [hunk 4]
```diff
@@ -0,0 +1,45 @@
+package dynamodbcopy
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+)
+
+type DynamoDBService interface {
```
Comment (raffo0707) [inline dynamodb.go]: missing comment

File: dynamodb.go [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+	UpdateCapacity(read, write int64) error
+}
+
+type dynamoDB struct {
+	tableName string
+	profile   string
+	api       *dynamodb.DynamoDB
+}
+
+func NewDynamoDB(tableName string, profile string) DynamoDBService {
```
Comment (raffo0707) [inline dynamodb.go]: also

**Comment by hfcorreia** [comment]
@raffo0707 I'll add the docs later, before creating the first release.

**Pull request merged**